combat-trainer.lic

```diff
@@ -1268,6 +1268,7 @@ class TrainerProcess
     pause
     waitrt?
     fix_standing
+    game_state.blessed_room
```

Comment:
So currently - no.  It will get set to true when you do meraud commune - and it will forever be true for the remainder of the hunt.  But the intent here was to have it be ready for another update - (I could bundle them I suppose) that will add a flag for the line which indicates meraud commune has worn off ('you no longer feel connected to your consecrated area') or what-have-you.  I would need to do a lil more testing in that area but when that's done then it could turn the @blessed_room back to false.

It should be noted there are absolutely no negatives to doing last_rites/recite in a non-blessed room or against constructs/undead/cursed creatures.  I went about it this way mainly because I just wanted to avoid repetitive actions that have no 'benefit'.